tools/please_go_install/BUILD

```diff
@@ -1,11 +1,28 @@
+#TODO(jpoole): once bugs are fixed, make this use the please_go_install in ~/.please
+package(FF_PLEASE_GO_INSTALL = False)
```

Comment:
Actually these should probably use go_get() otherwise it will break the bootstrap story